server/lib/database/src/postgres_m.erl

```diff
@@ -1,25 +1,40 @@
 -module(postgres_m).
 
 -behaviour(monad).
+
 -export(['>>='/2, return/1, fail/1]).
 
--type(monad(_A) :: {{'ok', any(), any()}, 'select'} 
-		 | {{'ok', any()}, 'delete'}
-		 | {{'ok', any()}, 'insert'}
-		 | {{'ok', any(), any(), any()}, 'insert'}
-		 | {{'ok', any()}, 'update'}
-		 | {{'error', any()}, any()}).
--include_lib("erlando/include/monad_specs.hrl").
+% This only makes sense if we export it or use dialyzer
+%-type monad(_A) ::
+%    {{ok, any(), any()}, select} |
+%    {{ok, any()}, delete} |
+%    {{ok, any()}, insert} |
+%    {{ok, any(), any(), any()}, insert} |
+%    {{ok, any()}, update} |
+%    {{error, any()}, any()}.
```

Comment:
Could you elaborate why this would only make sense if we export it? I would like to keep it in order to keep it consistent on how the original `erlando` does its monads.

Reply:
I need to know which functions use this otherwise it's never used by the functions inside the module (it never typed anything) or outside it.

If you want to use outside it, we can export the type btw.

Reply:
I don't follow why this needs to be used somewhere to be uncommented. My intention is to keep it in order to maintain the pattern created by `erlando`. In this sense, keeping it in form of a comment is ok.

Reply:
@EduardoLR10  then why not import `monad.monad`? the lib actually exports the type.

Reply:
I don't know, I actually haven't thought of that. I guess we could do that in the files using the custom monad.